src/main/resources/templates/login.html

```diff
@@ -18,10 +18,10 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-heading-examp
                         You have been logged out.
                     </div>
 
-
                     <h1 class="heading-large">Sign in</h1>
 
-                    <a href="/reset">Forgotten your password?</a>
+                    <p>If you do not have an account, or have forgotten your password, please
+                        <a class="lpg-get-in-touch" href="mailto:feedback@cslearning.gov.uk">get in touch to create one</a>.</p>
```

Comment:
Are we ok to have "_or have forgotten your password_" here considering we have a link for resetting below? Just thinking that reset request emails going to `feedback@cslearning` might be problematic if whoever is responsible for it doesn't have ability to reset.

Reply:
Good catch - updated.